package.json

```diff
@@ -74,19 +70,22 @@
     "tar-fs": "2.1.0"
   },
   "scripts": {
-    "build": "cross-env NODE_ENV=production npx calypso-build",
+    "build": "npm run build:assets && npm run archive",
+    "build:assets": "cross-env NODE_ENV=production npx calypso-build",
     "build:docs": "rm -rf hookdocs/ && jsdoc -c hookdoc-conf.json",
+    "build:vendor": "cp ./node_modules/select2/dist/css/select2.min.css ./node_modules/select2/dist/js/select2.full.js ./node_modules/select2/dist/js/select2.full.min.js ./assets/vendor/select2",
+    "archive": "composer archive --file=$npm_package_name --format=zip",
+    "postarchive": "rm -rf $npm_package_name && unzip $npm_package_name.zip -d $npm_package_name && rm $npm_package_name.zip && zip -r $npm_package_name.zip $npm_package_name && rm -rf $npm_package_name",
```

Comment:
Does this bit in `postarchive` do anything? Looks like it just unzips & zips it backs.

Reply:
I _think_ what it does is ensures the files are in a root directory with the plugin name for consistency in how they get unzipped.

So instead of `includes/file.php, blah.php` it is `sensei-lms/includes/file.php, blah.php`. I think without that there is some weird variations in how different unzipper's unzip (often using the filename of the zip). This way the zip could be renamed to `sensei-lms-beta2.zip` but it would unzip into `sensei-lms`.